builder/dockerfile/builder.go

```diff
@@ -218,7 +225,7 @@ func (b *Builder) build(stdout io.Writer, stderr io.Writer, out io.Writer) (stri
 		for k, v := range b.options.Labels {
 			line += fmt.Sprintf("%q=%q ", k, v)
 		}
-		_, node, err := parser.ParseLine(line)
+		_, node, err := parser.ParseLine(line, &b.directive)
```

Comment:
If we remove parsing from here(just creating `Node` objects here would be much safer and faster), can we remove the `directive` property of the builder and just implement it internally inside `Parse` method?